pkg/controller/certificatesigningrequest/helpers.go

```diff
@@ -80,8 +79,8 @@ func isNodeServingCert(csr *capi.CertificateSigningRequest, x509cr *x509.Certifi
 		log.Println("Usage does not match")
 		return false
 	}
-	if !strings.HasPrefix(x509cr.Subject.CommonName, "system:node:") {
-		log.Printf("CN does not match: %s\n", x509cr.Subject.CommonName)
+	if csr.Spec.Username != x509cr.Subject.CommonName {
+		log.Println("x509 CN %q doesn't match CSR username %q", x509cr.Subject.CommonName, csr.Spec.Username)
```

Comment:
Shouldn't this be `Printf` instead of `Println`, `Println` does not take formatting directions.

Reply:
:grimacing: thanks for catching that, will update shortly!